src/main/java/org/example/pageobject/pages/CartPage.java

```diff
@@ -0,0 +1,42 @@
+package org.example.pageobject.pages;
+
+import org.checkerframework.checker.units.qual.C;
+import org.example.pageobject.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class CartPage extends BasePage {
+    @FindBy(xpath = "//input[@value='Delete']")
+    WebElement delete;
+    @FindBy(xpath = "//div[@class=\"a-row sc-cart-header\"]/div/h1")
+    WebElement cartMessage;
+    @FindBy(xpath = "//span[@id='sc-subtotal-amount-activecart']/span")
+    WebElement price;
+    public CartPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public CartPage deleteItem(){
+        delete.click();
+        return new CartPage(webDriver);
+    }
+    public String getCartMessage(){
+        return cartMessage.getText();
+    }
+
+    public CartPage waitForPageLoad() {
```

Comment:
This technically isn't waiting for page load, it's waiting for the appearance of the Shopping Cart status text

Reply:
Changed the name of the method to "waitForCartStatusText"